src/mysql_lib/test.rs

```diff
@@ -0,0 +1,686 @@
+#[cfg(test)]
+mod tests {
+    use std::ops::Add;
+    use std::sync::Once;
+    use std::time::Duration;
+
+    use poise::serenity_prelude::{ChannelId, GuildId, MessageId, RoleId, UserId};
+    use serial_test::serial;
+    use sqlx::{MySql, Pool};
+    use sqlx::types::time::{Date, PrimitiveDateTime, Time};
+    use time::Month;
+
+    use crate::mysql_lib::{DatabaseAlumniRole, DatabaseGuild, DatabaseSemesterStudyGroup, DatabaseStudyGroup, DatabaseStudySubjectLink, DatabaseSubject, DatabaseTmpc, DatabaseTmpcJoinChannel, DatabaseTokenMessage, delete_all_token_messages, delete_alumni_role, delete_guild, delete_semester_study_group, delete_study_group, delete_study_subject_link, delete_subject, delete_tmpc, delete_tmpc_join_channel, delete_token_message, get_alumni_roles, get_connection, get_guild, get_semester_study_groups_in_guild, get_semester_study_groups_in_study_group, get_study_groups, get_study_subject_links_for_study_group, get_study_subject_links_for_subject, get_study_subject_links_in_guild, get_subjects, get_tmpc, get_tmpc_join_channel, get_token_messages, insert_alumni_role, insert_guild, insert_semester_study_group, insert_study_group, insert_study_subject_link, insert_subject, insert_tmpc, insert_tmpc_join_channel, insert_token_message, is_guild_in_database, migrate_database, update_tmpc_delete_at, update_alumni_role, update_alumni_role_separator_role, update_bot_channel, update_debug_channel, update_friend_role, update_ghost_enabled, update_ghost_kick_deadline, update_ghost_time_to_check, update_ghost_warning_deadline, update_help_channel, update_logger_pipe_channel, update_moderator_role, update_newsletter_role, update_nsfw_role, update_studenty_role, update_study_group, update_study_group_category, update_study_role_separator_role, update_subject_group_category, update_subject_role_separator_role, update_tmp_studenty_role, update_tmpc_keep_time, update_tmpc_join_channel_persist, update_tmpc_token, update_tmpc_keep};
```

Comment:
could just be shortened to `use crate::mysql_lib::*;`

Reply:
thx didn't think that was possible since the formatter didn't do it, will change